optimade/server/models/jsonapi.py

```diff
@@ -65,6 +69,15 @@ class Relationship(BaseModel):
     data: Optional[Union[Linkage, Set[Linkage]]]
     meta: Optional[dict]
 
+# class Empty(None):
+#     """Describes an empty to-one relationship."""
+
+class RelationshipToOne(Linkage):
+    """References to other resource objects in a to-one (\"relationship\"). Relationships can be specified by including a member in a resource's links object."""
+
+class RelationshipToMany(Set[Linkage]):
+    """An array of objects each containing \"type\" and \"id\" members for to-many relationships."""
```

Comment:
Just a question: is it necessary to escape quotation marks here?
Does it otherwise break the JSON?
I would have thought pydantic could be a bit more clever...